user_manager/api/urls.py

```diff
@@ -8,5 +8,5 @@
 from .v1 import urls
 
 urlpatterns = [
-    url(r'^v1/', include((urls, 'user_manager'), namespace='v1')),
+    url(r'^v1/', include(urls, namespace='v1')),
```

Comment:
Side note: The old syntax here is very opaque.  Thanks for cleaning it up.

Reply:
The bummer is, it's actually the [new syntax](https://docs.djangoproject.com/en/2.0/_modules/django/urls/conf/#include) for specifying the `app_name` with the URLs!  The Django 1.8 code let you specify them as kwargs, but now, we can't.  Ucky!

Reply:
Dang it.